Neos.ContentGraph.DoctrineDbalAdapter/src/ContentGraphReadModelAdapter.php

```diff
@@ -45,26 +47,44 @@ public function __construct(
     ) {
     }
 
-    public function buildContentGraph(WorkspaceName $workspaceName, ContentStreamId $contentStreamId): ContentGraph
-    {
-        return new ContentGraph($this->dbal, $this->nodeFactory, $this->contentRepositoryId, $this->nodeTypeManager, $this->tableNames, $workspaceName, $contentStreamId);
-    }
-
-    public function findWorkspaceByName(WorkspaceName $workspaceName): ?Workspace
+    public function getContentGraph(WorkspaceName $workspaceName): ContentGraph
```

Comment:
not your original sin, but like this is the exact problem iwt the find/get thing. This does a DB query every time, so it hsould be more like find? 🤔 

Either way I think we should do something about this, as getContentGraph() is used ALL over the place. This shouldn't be a query everytime.

Reply:
added todo to readd cache